contrib/pom.xml

```diff
@@ -59,6 +59,7 @@
     <module>format-pcapng</module>
     <module>format-iceberg</module>
     <module>format-deltalake</module>
+    <module>format-daffodil</module>
```

Comment:
Please keep these in alphabetical order.

Reply:
@mbeckerle #2825 is merged, please rebase on master when convenient.